modules/nw-networkpolicy-delete.adoc

```diff
@@ -75,3 +76,10 @@ ifdef::multi[]
 endif::multi[]
 :!name:
 :!role:
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role in the console, then you
```

Comment:
```suggestion
If you log in with `cluster-admin` permissions in the web console, then you
```

Just a suggestion for easier reading. I would apply this change to the other NOTE below.